source/functions/eulXYZ2RotMat.m

```diff
@@ -0,0 +1,22 @@
+function rotMat = eulXYZ2RotMat(phi, theta, psi)
```

Comment:
This is probably a broader point...but have we decided on a naming convention for functions?

My personal preference would be eulxyz_2_rotmat(), but I think I used eulXYZ2RotMat() originally in keeping with the rest of the code. But IIRC there were mixed conventions and we wanted to resolve that problem? Are we going for camel case function names?

@kmruehl

Reply:
in fact, the orbital_velocity() function below has an underscore. I would prefer underscores for functions, camel case for variables. Let me know what you think 👍

Reply:
I agree the naming convention is a bit inconsistent, especially with variables. I think this is a broader cleanup item on the project board. 

I think we have waited to address this so far because there were concerns about backwards compatibility if a lot of the class variables are renamed

Reply:
yeah that's a good point, it will take time to propagate the convention throughout the whole code. But perhaps this PR is a good moment to decide on a convention? (If we haven't already)

Reply:
@dav-og I was going to bring up the convention today, but ran out of time.

How about we leave these camel case for now? The rest of the functions in /source/functions are currently camel case. Though I agree, we should have a different convention for variables, functions, classes. Maybe the team can discuss during the working meeting.

Reply:
I would love to see some kind of coding guidelines introduced. It would be great to see a Python style line length restriction, as some of the lines are way too long at the moment. Consistent naming style is great too - I'm not sure if you necessarily need to distinguish every type of variable, but consistency helps a lot.

I wouldn't recommend implementing this in a big bang style, however, as you won't be able to properly review such a huge change. This kind of thing is best implemented as other changes are being made, at least until there is very little left to modify.

I think someone should open an issue to discuss it. We might need a doodle poll to vote on the options!

Reply:
Another thing is you can get a handle on the "accepted" naming conventions from the MATLAB docs. E.g:

* [Classes, classMethods and Properties](https://uk.mathworks.com/help/matlab/matlab_oop/create-a-simple-class.html)
* [functionDefinitions](https://uk.mathworks.com/help/matlab/ref/function.html#bvfsr6o)
* [variables_whoKnows](https://uk.mathworks.com/help/matlab/matlab_prog/variable-names.html)

There isn't any clarity on how a variable should be named. Personally I prefer UPPERCASE GLOBALS, for instance.

Reply:
@dav-og and @akeeste we decided on camel case for WEC-Sim a long time ago (many years ago), and I think we should stick with it. Yes, we do have functions/variables that did not follow this naming convention, and we should try to clean them up this year as we we come across them. This is the general cleanup on the project board @akeeste referred do.